packages/udaru-core/lib/ops/authorizeOps.js

```diff
@@ -19,6 +19,20 @@ function badImplementationWrap (next) {
   }
 }
 
+function getContext (params) {
```

Comment:
small comment on naming: this looks more like a `buildContext`

Reply:
will change :-)